service/history/workflowResetter_test.go

```diff
@@ -31,36 +31,37 @@ import (
 
 	"github.com/uber/cadence/.gen/go/shared"
 	"github.com/uber/cadence/common"
-	"github.com/uber/cadence/common/clock"
 	"github.com/uber/cadence/common/collection"
 	"github.com/uber/cadence/common/definition"
 	"github.com/uber/cadence/common/log"
 	"github.com/uber/cadence/common/log/loggerimpl"
+	"github.com/uber/cadence/common/metrics"
 	"github.com/uber/cadence/common/mocks"
 	"github.com/uber/cadence/common/persistence"
+	"github.com/uber/cadence/common/resource"
 )
 
 type (
 	workflowResetterSuite struct {
 		suite.Suite
 		*require.Assertions
 
-		mockShard           *shardContextImpl
-		mockClusterMetadata *mocks.ClusterMetadata
-		mockHistoryV2Mgr    *mocks.HistoryV2Manager
-		mockStateRebuilder  *MocknDCStateRebuilder
-		logger              log.Logger
-
 		controller         *gomock.Controller
+		mockResource       *resource.Test
 		mockTransactionMgr *MocknDCTransactionMgr
 
-		workflowResetter *workflowResetterImpl
+		mockShard          *shardContextImpl
+		mockHistoryV2Mgr   *mocks.HistoryV2Manager
+		mockStateRebuilder *MocknDCStateRebuilder
```

Comment:
could you put all `*gomock.Controller` related vars in the section under `controller         *gomock.Controller`?

Reply:
Good catch. Missed this one.